docs/pages/bare/exploring-bare-workflow.md

```diff
@@ -21,7 +21,7 @@ If you already have a React Native project that has been created with `react-nat
 
 ### Existing Expo managed workflow apps
 
-If you already have an Expo managed workflow app and you need to customize the native code, you can eject to the bare workflow by running `expo eject`. This will give you a vanilla React Native app that includes all of the Expo SDK APIs that you were using already, and no more than that. The outcome is that you will be in just as good of a position as if you had started your app in the bare workflow from scratch, only you probably saved yourself some time!
+If you already have an Expo managed workflow app and you need to customize the native code, you can eject to the bare workflow by running `expo prebuild`. This will give you a vanilla React Native app that includes all of the Expo SDK APIs that you were using already, and no more than that. The outcome is that you will be in just as good of a position as if you had started your app in the bare workflow from scratch, only you probably saved yourself some time!
```

Comment:
This paragraph is not inline with the spirit of the linear task.

Reply:
What's your suggestion?